public/js/app/pages/Start.jsx

```diff
@@ -1,29 +1,13 @@
 import React, { Component, Fragment } from 'react'
 import { inject, observer } from 'mobx-react'
+import Button from '../components/Button'
 
 @inject(['routerStore'])
 @observer
 class Start extends Component {
-  state = {
-    buttonClicked: false,
-  }
-
-  toggleButton = () => this.setState({ buttonClicked: !this.state.buttonClicked })
-
   render() {
     const { message } = this.props.routerStore
-    return (
-      <Fragment>
-        <h1 className="display-3">Node-web</h1>
-        <h2>You are upp and running kth-node react!</h2>
-        <hr className="my-2" />
-        <h3>{`Message from routerStore: ${message}`}</h3>
-        <button type="button" onClick={this.toggleButton}>
-          Try me
-        </button>
-        <p>{this.state.buttonClicked ? 'Button works!' : ''}</p>
-      </Fragment>
-    )
+    return <Button message={message} />
```

Comment:
Jag själv tycker egentligen om att använda "class" i samband med React-komponenter. Men mångra föredrar functional components nu, tror jag. Kanske skulle man ändra det här till en funktion - speciellt så länge det inte behövs hooks.

Förmodligen måste man då ta bort decorators och exporterar något som ```export default inject(['routerStore'])(observer(Start))```.

Reply:
Väntar tills vi mergat in era fina uppdateringar. Just nu behövs state.